cluster-autoscaler/FAQ.md

```diff
@@ -566,27 +570,41 @@ Provisioning Request (abbr. ProvReq) is a new namespaced Custom Resource that ai
 
 #### Enabling ProvisioningRequest Support
 
-1. **Cluster Autoscaler Version**: Ensure you are using Cluster Autoscaler version 1.30.1 or later.
+1. __Cluster Autoscaler Version__: Ensure you are using Cluster Autoscaler version 1.30.1 or later.
 
-2. **Feature Flag**: Enable ProvisioningRequest support by setting the following flag in your Cluster Autoscaler configuration:
+2. __Feature Flag__: Enable ProvisioningRequest support by setting the following flag in your Cluster Autoscaler configuration:
 --enable-provisioning-reques=true.
 
 #### Supported ProvisioningClasses
 
 Currently, ClusterAutoscaler supports following ProvisioningClasses:
 
-* `check-capacity.autoscaling.x-k8s.io`. 
+* `check-capacity.autoscaling.x-k8s.io`.
 When using this class, Cluster Autoscaler performs following actions:
 
-  * **Capacity Check**: Determines if sufficient capacity exists in the cluster to fulfill the ProvisioningRequest.
+  * __Capacity Check__: Determines if sufficient capacity exists in the cluster to fulfill the ProvisioningRequest.
 
-  * **Reservation from other ProvReqs** (if capacity is available): Reserves this capacity for the ProvisioningRequest for 10 minutes, preventing other ProvReqs from using it.
+  * __Reservation from other ProvReqs__ (if capacity is available): Reserves this capacity for the ProvisioningRequest for 10 minutes, preventing other ProvReqs from using it.
 
-  * **Condition Updates**:
+  * __Condition Updates__:
   Adds a Accepted=True condition when ProvReq is accepted by ClusterAutoscaler and ClusterAutoscaler will check capacity for this ProvReq.
   Adds a Provisioned=True condition to the ProvReq if capacity is available.
   Adds a BookingExpired=True condition when the 10-minute reservation period expires.
 
+* `best-effort-atomic-scale-up.autoscaling.x-k8s.io` (supported from Cluster Autoscalere version 1.30.2 or later).
+When using this class, Cluster Autoscaler performs following actions:
+
+  * __Capacity Check__: Check which pods could be scheduled on existing capacity.
+
+  * __ScaleUp Request__: Evaluates if scaling up a node group could fulfill the remaining requirements of the ProvisioningRequest..
+
+  * __Reservation from other ProvReqs (if scale up request succeeded)__: Reserves this capacity for the ProvisioningRequest for 10 minutes, preventing other ProvReqs from using it.
+
+  * __Condition Updates__:
+  Adds a Accepted=True condition when ProvReq is accepted by ClusterAutoscaler.
```

Comment:
looks good to me, just a small nit, can we mention these conditions in a numbered list, because currently it is showing in a paragraph?

Reply:
Something like this:
1.  Adds a Accepted=True condition when ProvReq is accepted by ClusterAutoscaler.
2.  Adds a Provisioned=True condition to the ProvReq if the node group scale up request is successful.
3.  Adds a BookingExpired=True condition when the 10-minute reservation period expires.

@yaroslava-serdiuk, WDYT?

Reply:
Good point, thanks! Markdown adds `a, b, c` instead of numbers, so I added bullet points instead.

Reply:
> Markdown adds a, b, c instead of numbers, so I added bullet points instead.

@yaroslava-serdiuk, The changes look good to me.